src/role_quester.js

```diff
@@ -16,7 +16,7 @@ roles.quester.settings = {
 roles.quester.questLost = function(creep, quest, reason, value) {
   creep.log(`Quest lost cs: ${value} ${JSON.stringify(quest)}`);
   delete Memory.quests[creep.memory.level];
-  creep.suicide();
+  Creep.recycleCreep(creep);
```

Comment:
maybe we should add into `Creep.recycleCreep` function 
`creep.memory.timeToTravel > 0 && creep.memory.born - Game.time < creep.memory.timeToTravel` and let the recycleCreep decide between suicide and recycle

Reply:
> maybe we should add into `Creep.recycleCreep` function `creep.memory.timeToTravel > 0 && creep.memory.born - Game.time < creep.memory.timeToTravel` and let the recycleCreep decide between suicide and recycle

yes! that is what I mean